Assets/HoloToolkit/Input/Scripts/Editor/SpeechInputHandlerEditor.cs

```diff
@@ -13,16 +13,21 @@ public class SpeechInputHandlerEditor : Editor
     {
         private SerializedProperty keywordsProperty;
         private string[] registeredKeywords;
+        private SerializedProperty isGlobalListenerProperty;
+        private bool isGlobalListerer;
```

Comment:
Whoops, first time I wrote an editor. That was a leftover from a failed first attempt. Also, take heed: by setting IsGlobalListener to true by default I changed the default behavior of the SpeechInputHandler. I modified my PR to set that default to false, so you actually have to consciously make your SpeechInputHandler a global listener. This is to prevent people rebuilding the sample and inadvertently ending up with a lot of global listeners